src/rubrix/client/sdk/metrics/api.py

```diff
@@ -47,7 +48,7 @@ def calculate_metric(
         headers=client.get_headers(),
         cookies=client.get_cookies(),
         timeout=client.get_timeout(),
-        json={},  # TODO: parameterize !!!
+        json={"query_text": query},
```

Comment:
does this fail if `query=None` ?

Reply:
API accepts these kind of request. For query=None, no filters at all will be applied

Reply:
:+1: